services/catalog/src/simcore_service_catalog/core/background_tasks.py

```diff
@@ -134,16 +95,15 @@ async def _ensure_registry_and_database_are_synced(app: FastAPI) -> None:
 
     Notice that a services here refers to a 2-tuple (key, version)
     """
-    services_in_registry: dict[
-        tuple[ServiceKey, ServiceVersion], ServiceMetaDataPublished
-    ] = await _list_services_in_registry(app)
+    director_api = get_director_api(app)
+    services_in_manifest_map = await manifest.get_services_map(director_api)
```

Comment:
Here it's slightly better in the sense that you map a service to a manifest. Personally I would have picked `to` instead of `in`.